docker/php/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM php:7.4.3-fpm-alpine
+
+RUN apk add pcre-dev ${PHPIZE_DEPS} \
+  && pecl install xdebug \
+  && docker-php-ext-enable xdebug \
+  && apk del pcre-dev ${PHPIZE_DEPS}
```

Comment:
Do we need docker for unit testing? We are testing 2 files and there is no need for a complicated setup with databases, hosts and such which would be the main reason for using docker. I would vote for removing docker.

Reply:
So to run the unit tests, you'd need php installed on your system. I don't have it for example and I wouldn't like to install it.

Reply:
Yea but this is a vendor package, you could just add the tests to their composer.json and run it in whatever server or container they use.

Reply:
Alright that's valid. I shut up now.

Reply:
I was also undecided if enabling docker support or not, @ValterJunior idea was not to use it, @juliano-barros suggested to add it so it will be easier for maintenance. In my opinion having a self-hosted repository that needs as little configuration as possibile is a plus, especially team-wise.